usr/local/www/diag_resetstate.php

```diff
@@ -43,21 +43,31 @@
 ##|*MATCH=diag_resetstate.php*
 ##|-PRIV
 
+define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
```

Comment:
It allows me to test the appearance and functionality of the page without actually deleting the state tables and so having to wait until the firewall has built some more.

Like all DEBUG stuff, it will be set to false before release.

Steve

> On May 8, 2015, at 9:44 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_resetstate.php <https://github.com/SjonHortensius/pfsense/pull/154#discussion_r29940190>:
> 
> > @@ -43,21 +43,31 @@
> >  ##|*MATCH=diag_resetstate.php*
> >  ##|-PRIV
> >  
> > +define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
> I understood the DEBUG in reboot.php; but I'm not sure why it's here
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/154/files#r29940190>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>